go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/gardener/etcd-druid v0.15.3
-	github.com/gardener/gardener v1.66.1
-	github.com/gardener/machine-controller-manager v0.45.0
+	github.com/gardener/gardener v1.70.2
+	github.com/gardener/machine-controller-manager v0.48.1
```

Comment:
is it also required to update the equinix machine prover: https://github.com/gardener/machine-controller-manager-provider-equinix-metal ?

Reply:
> is it also required to run gardener 1.70 as minimum supported version?

No.

> is it also required to update the equinix machine prover

Not with this version of Gardener. But for future versions of gardener ref- https://github.com/gardener/gardener/pull/7599#issuecomment-1459398514, MCM-provider will be required  to vendor new version for MCM.